articles/active-directory-b2c/active-directory-b2c-setup-aad-custom.md

```diff
@@ -50,6 +50,9 @@ To enable sign-in for users from a specific Azure AD organization, you need to r
 1. Select **Web app / API** for the application type.
 1. For **Sign-on URL**, enter the following URL, where `yourtenant` is replaced by the name of your Azure AD B2C tenant (`fabrikamb2c.onmicrosoft.com`):
 
+>[!NOTE]
+> The value for "yourtenant" must be all lowercase in the Sign-on URL regardless of the case used within B2C.
```

Comment:
@martinbherbener Thanks for adding this! Could you update it to say "The value for "yourtenant" must be all lowercase in the Sign-on URL" (just remove the ending)
And also indent it one right?